### PR TITLE
Update snapcraft.yaml

### PR DESCRIPTION
**Pull request by nskaggs**
Change to classic snap format

### CODE REVIEW

**Comment by nskaggs** [comment]
!!build!!

**Comment by nskaggs** [comment]
!!build!!

File: snapcraft.yaml [hunk 1]
```diff
@@ -16,10 +16,11 @@ parts:
     source: https://github.com/juju/juju.git
     source-type: git
     build-packages: [gcc]
-    #If you want to grab a specific revision tag, include it here
```
Comment (jameinel) [inline snapcraft.yaml]: Similarly here, if I'm developing a feature branch, and I want to use my CWD, is there a way to trigger that?

Comment (nskaggs) [inline snapcraft.yaml]: Yes, you could make the source local instead. Including an example of this would be useful.

File: snapcraft.yaml [hunk 2]
```diff
@@ -16,10 +16,11 @@ parts:
     source: https://github.com/juju/juju.git
```
Comment (nskaggs) [inline snapcraft.yaml]: Same as below. Perhaps it should be the default for this snapcraft.yaml checked into the repo.

**Comment by nskaggs** [comment]
I'll re-propose in a new PR.

**Pull request closed without merging**